packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -112,7 +133,7 @@ export class ServerDeploymentGroup extends ServerDeploymentGroupRef {
     public readonly deploymentGroupName: DeploymentGroupName;
 
     constructor(parent: cdk.Construct, id: string, props?: ServerDeploymentGroupProps) {
```

Comment:
Instead of having to null-check props everywhere, you can assign a default value of {}

Reply:
Sure.